docs/fundamentals/networking/telemetry/tracing.md

```diff
@@ -0,0 +1,169 @@
+
+> [!NOTE]
+> The terms *span* and *activity* are being used as synonyms in this article. In context of .NET code, they refer to a <xref:System.Diagnostics.Activity?displayProperty=nameWithType> instance. The OTel span isn't to be confused with <xref:System.Span%601?displayProperty=nameWithType>.
+
+> [!TIP]
+> For a comprehensive list of all built-in activities together with their tags/attributes, see [Built-in activities in .NET](../../../core/diagnostics/distributed-tracing-builtin-activities.md).
+
+## Instrumentation
+
+In order to emit traces, the `System.Net` libraries are [instrumented](../../../core/diagnostics/distributed-tracing-instrumentation-walkthroughs.md#add-basic-instrumentation) with built-in <xref:System.Diagnostics.ActivitySource>-s which create <xref:System.Diagnostics.Activity> objects to track the work performed. Activities are only created if there are listeners subscribed to the <xref:System.Diagnostics.ActivitySource>.
+
+The built-in instrumentation has evolved with .NET versions.
+
+- On .NET 8 and earlier, the instrumentation is limited to the creation of an empty [HTTP client request activity](../../../core/diagnostics/distributed-tracing-builtin-activities.md#http-client-request). This means that users have to rely on the [`OpenTelemetry.Instrumentation.Http`](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/1ca05685cbad63d3fa813b9cab49be341048e69e/src/OpenTelemetry.Instrumentation.Http#httpclient-and-httpwebrequest-instrumentation-for-opentelemetry) library to populate the activity with the information (for example, tags) needed to emit useful traces.
+- .NET 9 extended the instrumentation with the emission the name, status, exception info, and the most important tags according to the OTel [HTTP client semantic conventions](https://opentelemetry.io/docs/specs/semconv/http/http-spans/#http-client) on the HTTP client request activity. This means that on .NET 9+ the `OpenTelemetry.Instrumentation.Http` dependency can be omitted, unless more advanced features like [enrichment](#enrichment) are required.
+- .NET 9 also introduced [experimental connection tracing](#experimental-connection-tracing), adding new activities across the `System.Net` libraries to support diagnosing connection issues.
+
+## Collect System.Net traces
```

Comment:
This section and the sections on Aspire seem to be exactly the same as some sections in metrics.md. Is that intended?

Reply:
One is a walkthrough on metrics the other one is a walkthrough on tracing. There is a lot of duplicate content since both walktroughs present the same tools (Aspire & Service Defaults). I don't see how (and why) would we dedupe this.